packages/zoe/src/contracts/loan/updateDebt.js

```diff
@@ -1,8 +1,12 @@
 // @ts-check
 
 import '../../../exported';
-import { makeNotifierKit } from '@agoric/notifier';
-import { E } from '@agoric/eventual-send';
+import {
+  makeNotifierKit,
+  // After #2511, replace with observeNotifier with swapped arguments
+  updateFromNotifier,
```

Comment:
Sorry I forgot to do that in my PR-on-PR. But please do that before merging
```suggestion
  observeNotifier,
```

Reply:
I see it is done. Resolving.